### DISCUSSION TITLE
two column option for file browser?

### DISCUSSION THREAD

**Discussion by zampanoo** [koreader/koreader#11547] [Q&A]
I am using koreader on a 13.3 e-ink device and I like it. When using the file browser in the classic view (which I prefer), a lot of space is not used at all: the folder and file names sit on the left of the screen, and some matadata is displayed on the right of the screen - and acres of unused white space in between. 

I wonder whether there is an option to split the screen into two (or more) columns? It would be possible to display much more information that way. 

RelaunchX, for example, manages to arrange the same information in three columns, displaying only the file or folder names. Hence, I do not need to visit two other pages (as in koreader) to access the files and folders. 

Any pointers welcome.

**Comment by Frenzie** [comment]
There are some options for the cover display setting. I wouldn't know without looking at the code, but perhaps it's not overly complicated to allow something similar with the list view, although I guess you'd have to put in some framecontainers… maybe @poire-z or @hius07 has a better idea of just how much work it'd be. In any case, see plugins/coverbrowser for the relevant code if you want to try some hacking.
<img src=https://github.com/koreader/koreader/assets/202757/681941f0-79a8-407e-a70e-d14850a44599 width=40%>

**Reply by poire-z** [reply]
Dunno, really. Probably much work for some edge use case.
And we shouldn't spend much time in File browser, this software is for reading, not contemplating one's book shelves :)

**Reply by zampanoo** [reply]
Thanks, Frenzie: I am a just a user, so won't be able to edit code, unfortunately.

... and poire-z: Well, exactly, you nailed it - if more of my folders and and files are visible on a single page in list view (rather than on two or three pages), I would certainly spend less time in the file browser move back and forth :-)  With dozens of folders and lots of academic pdf files, a space-saving two-column list view would be very handy.
